src/functionalTest/java/uk/gov/hmcts/reform/orgrolemapping/befta/OrgRoleMappingAmTestAutomationAdapter.java

```diff
@@ -66,8 +68,18 @@ public Object calculateCustomValue(BackEndFunctionalTestScenarioContext scenario
                 return null;
             case ("tomorrow"):
                 return LocalDate.now().plusDays(1);
+            case ("generateS2STokenForOrm"):
+                return new TokenUtils().generateServiceToken(buildOrmSpecificConfig());
             default:
                 return super.calculateCustomValue(scenarioContext, key);
         }
     }
+
+    private UserTokenProviderConfig buildOrmSpecificConfig() {
+        UserTokenProviderConfig config = new UserTokenProviderConfig();
+        config.setMicroService("am_org_role_mapping_service");
+        config.setSecret(System.getenv("AM_ORG_S2S_SECRET"));
```

Comment:
I think this needs to be `AM_ORG_ROLE_MAPPING_SERVICE_SECRET`.

Reply:
Updated the review comment.